### PR TITLE
Add stride testnet two instructions

### PR DESCRIPTION
**Pull request by asalzmann**

### CODE REVIEW

File: replicated-security/stride-ics-testnet-two-1/readme.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+| 2 | Now until software upgrade proposal passes on Stride | | Build (or download) the target (post-transition) Stride binary. <br><br>If you are using Cosmovisor, place place it in Cosmovisor `/upgrades/<upgrade-name>/bin` directory.<br><br>If you are not using Cosmovisor, be ready to manually switch your binary at the upgrade halt height. |
+| 3 | Now until spawn time (15:00 UTC) | Submit assign-consensus-key for stride-ics-testnet-two-1 with the keys on your full node (**note: make sure to do this before spawn time!)** | |
+| 4 | During voting period for  consumer-addition proposal on provider | You don’t have to do anything. Optionally, you may vote for the consumer-addition proposal | |
+| 5 | During voting period for software upgrade on Stride | | You don’t have to do anything. |
+| 6 | After spawn time | | Place the newly generated “genesis” file (containing only the ccv state) in the ⚠️ `$NODE_HOME/config directory` ⚠️ Stride will provide this.<br><br>Do NOT replace the existing genesis file. |
```
Comment (dasanchez) [inline replicated-security/stride-ics-testnet-two-1/readme.md:112]: This is different from the "Download new Sovereign genesis" step from the above instructions- is it `$NODE_HOME/config` or `$HOME/.sovereign/config`?

File: replicated-security/stride-ics-testnet-two-1/readme.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+Make directories in cosmovisor and copy binaries
+```
+mkdir -p $HOME/.stride/cosmovisor/upgrades/v11/bin/
+cp $HOME/go/bin/strided $HOME/.stride/cosmovisor/upgrades/v11/bin/
+```
+
+Download new Sovereign genesis
+```
+mkdir -p $HOME/.sovereign/config/
+wget -O $HOME/.sovereign/config/genesis.json https://cdn.discordapp.com/attachments/1064857924402413600/1116022613966336041/genesis_new.json
```
Comment (dasanchez) [inline replicated-security/stride-ics-testnet-two-1/readme.md:93]: Can you confirm the path will still be `$HOME/.sovereign/config/` ?
Also, we should not list a URL until the new genesis is available- putting this up ahead of the spawn time will probably lead to people downloading the first rehearsal's patched genesis file.

File: replicated-security/stride-ics-testnet-two-1/readme.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+git checkout 5a45a4cfaef54aa0e875eb4e618bfae882d278a2
+
+make install
+```
+
+replace the binary
+
+```sh
+mkdir -p $HOME/.sovereign/config
```
Comment (deniseyu) [inline replicated-security/stride-ics-testnet-two-1/readme.md:62]:
```suggestion
mkdir -p $NODE_HOME/config
```

File: replicated-security/stride-ics-testnet-two-1/readme.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+curl -L https://github.com/cosmos/testnets/raw/master/replicated-security/stride-ics-testnet-two-1/genesis.json -o $HOME/.sovereign/config/genesis.json
```
Comment (deniseyu) [inline replicated-security/stride-ics-testnet-two-1/readme.md:64]:
```suggestion
curl -L https://github.com/cosmos/testnets/raw/master/replicated-security/stride-ics-testnet-two-1/genesis.json -o $NODE_HOME/config/ccv.json
```

File: replicated-security/stride-ics-testnet-two-1/readme.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+```sh
+git clone https://github.com/Stride-Labs/stride.git
+cd stride
+git checkout a3eff2dc
+make install
+strided init stride-node --chain-id stride-ics-testnet-two-1
+
+# Grab the genesis file
+curl -L https://raw.githubusercontent.com/Stride-Labs/mainnet/ics-testnet/ics-testnet/genesis.json -o $HOME/.stride/config/genesis.json
```
Comment (deniseyu) [inline replicated-security/stride-ics-testnet-two-1/readme.md:44]:
```suggestion
curl -L https://raw.githubusercontent.com/Stride-Labs/mainnet/ics-testnet/ics-testnet/genesis.json -o $NODE_HOME/config/genesis.json
```

File: replicated-security/stride-ics-testnet-two-1/readme.md (hunk 3 above)
Comment (deniseyu) [inline replicated-security/stride-ics-testnet-two-1/readme.md:62]:
```suggestion
```

Comment (deniseyu) [reply replicated-security/stride-ics-testnet-two-1/readme.md:62]: We don't need to create this folder now, it's already there

Comment (deniseyu) [reply replicated-security/stride-ics-testnet-two-1/readme.md:62]: Actually I think we want to remove everything from line 59 to 65. Bosco's instructions below do a better job of describing the binary swap step

**Pull request merged**